setup.py

```diff
@@ -80,4 +80,9 @@ def find_version(*file_paths):
         'scikit-learn==0.23.2',
         'tqdm==4.54.1'],
     extras_require=extras,
+    entry_points={
+        "console_scripts": [
+            'omop2obo = omop2obo.main:main',
```

Comment:
if you change the name of the python file from `main.py` to `cli.py`, change this line like:
```suggestion
            'omop2obo = omop2obo.cli:main',
```